cabal.project

```diff
@@ -47,39 +45,14 @@ allow-newer:
   , servant-foreign:lens
   , blockfrost-api:lens
 
-constraints:
-  -- cardano-prelude-0.1.0.0 needs
-  , protolude <0.3.1
-
-  -- cardano-ledger-byron-0.1.0.0 needs
-  , cardano-binary <1.5.0.1
-
-  -- plutus-core-1.0.0.1 needs
-  , cardano-crypto-class >2.0.0.0
-  , algebraic-graphs <0.7
-
-  -- cardano-ledger-core-0.1.0.0 needs
-  , cardano-crypto-class <2.0.0.1
-
-  -- cardano-crypto-class-2.0.0.0.1 needs
-  , cardano-prelude <0.1.0.1
-
-  -- dbvar from cardano-wallet needs
-  , io-classes <0.3.0.0
-
-  -- newer typed-protocols need io-classes>=0.3.0.0 which is incompatible with dbvar's constraint above
-  , typed-protocols==0.1.0.0
-
-  , aeson >= 2
-
-  , hedgehog >= 1.1
-
-  , resource-pool <0.4.0.0
 
-  , http2 <4.0.0
-
-  -- ouroboros-consensus-shelley-0.1.0.1 needs
-  , ouroboros-consensus-protocol==0.1.0.1
+allow-older:
+  -- freer-extras works with lens-5.2 (rather than 5.2.1)
+  , freer-extras:lens
+  -- freer-extras works with resource-pool-0.3.1.0 (rather than 0.4.0.0)
+  , freer-extras:resource-pool
+  -- plutus-ledger works with lens-5.2 (rather than 5.2.1)
+  , plutus-ledger:lens
```

Comment:
This can be removed no? We don't depend on freer-extras or plutus-ledger.

Reply:
Yep, thanks!